config/routes.rb

```diff
@@ -45,4 +45,5 @@
   end
 
   resources :user_videos, only:[:create, :destroy]
+  get '/auth/github/callback', to: 'sessions#update'
```

Comment:
I think we made to correct choice setting this to the update action instead of the create action.